src/main/java/duke/tasks/Todo.java

```diff
@@ -0,0 +1,17 @@
+package duke.tasks;
+
+public class Todo extends Task {
+
+    public Todo(String description) {
+        super(description);
+        System.out.println("Got it. I've added this task:");
```

Comment:
Not that it is wrong but, is it possible to separate the print statements with the Task Classes themselves? So instead of printing here, you print in the caller function instead. It is best practice to separate the display and the data if possible. 

so for example 

Todo.java
```java 
public Todo(String description) {
    super(description);
}
```
Duke.java / wherever you create an object of this class
```
list.add(new Todo("blablabla");
System.out.println("Got it. I've added this task:");
System.out.println("  " + this);
System.out.println("Now you have " + super.size + " tasks in the list.");
```